HUD/GetMenuLayoutChangedEventDetails.md

```diff
@@ -0,0 +1,38 @@
+---
+ns: HUD
+aliases: ["0x7E17BE53E1AAABAF", "_GET_PAUSE_MENU_SELECTION_DATA"]
+---
+## GET_MENU_LAYOUT_CHANGED_EVENT_DETAILS
+
+```c
+// 0x7E17BE53E1AAABAF 0x6025AA2F
+void GET_MENU_LAYOUT_CHANGED_EVENT_DETAILS(int* oldMenuState, int* currentMenuState, int* selectedItemUniqueId);
+```
+
+Returns details of the current layout changed event. Also see the HasMenuLayoutChangedEventOccurred native.
+
+oldMenuState: This is the menuID of the last selected item minus 1000 (lastItem.menuID - 1000)
+currentMenuState: Same as oldMenuState except for the currently selected menu item
+selectedItemUniqueId: This is uniqueID of the currently selected menu item
+menu state values can be viewed here (or PauseMenuLUT.as in scaleform files): https://pastebin.com/ZiMpN3mN
```

Comment:
Using pastebin is usually frowned upon, you could put this on the forums or have @4mmonium put it into his gist

Reply:
Got you, I made a commit with my own gist for the time being but as soon as 4mmonium makes his, i'll change it